packages/pocket-ic/BUILD.bazel

```diff
@@ -256,3 +256,34 @@ rust_test(
         "@crate_index//:candid",
     ],
 )
+
+rust_test(
+    name = "canister_snapshots",
+    size = "medium",
+    srcs = ["tests/canister_snapshots.rs"],
+    data = [
+        "//:dfx",
+        "//packages/pocket-ic/test_canister:test_canister.wasm.gz",
+        "//rs/pocket_ic_server:pocket-ic-server",
+        "@mozilla_root_ca_store//file",
+    ],
+    env = {
+        "DFX": "$(rootpath //:dfx)",
+        "POCKET_IC_BIN": "$(rootpath //rs/pocket_ic_server:pocket-ic-server)",
+        "SSL_CERT_FILE": "$(rootpath @mozilla_root_ca_store//file)",
+        "TEST_WASM": "$(rootpath //packages/pocket-ic/test_canister:test_canister.wasm.gz)",
+    },
+    tags = [
+        "cpu:16",
+        # TODO: remove 'requires-network' tag when the root cause for sporadic error below on Apple Silicon is identified and fixed.
+        # Failed to crate http gateway: Failed to bind to address 127.0.0.1:0: Operation not permitted (os error 1)
+        "requires-network",
```

Comment:
Maybe I misremember but I seem to recall you removed this tag in some other PR. Do we still need this?